packages/react-reconciler/src/ReactTypeOfMode.js

```diff
@@ -12,8 +12,8 @@ export type TypeOfMode = number;
 export const NoMode = /*                         */ 0b0000000;
 // TODO: Remove ConcurrentMode by reading from the root tag instead
 export const ConcurrentMode = /*                 */ 0b0000001;
-export const ProfileMode = /*                    */ 0b0000010;
-export const DebugTracingMode = /*               */ 0b0000100;
+export const ProfileMode = /*                     */ 0b0000010;
+//export const DebugTracingMode = /*             */ 0b0000100; // Removed
```

Comment:
@sebmarkbage @acdlite when we delete a mode, should we shift the bits or is there a good reason to keep the slot (maybe DevTools needs to match up, for example idk)?

Reply:
Looks like DevTools use it so probably easiest leave it.

https://github.com/facebook/react/blob/main/packages/react-devtools-shared/src/backend/fiber/renderer.js#L334